src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java

```diff
@@ -111,7 +146,9 @@ protected JacobianMatrix createJacobianMatrix(EquationSystem equationSystem, Vol
         // Wrap factors in specific class to have instant access to their branch, and their equation term
         private final SensitivityFactor factor;
 
-        private final LfBranch functionLfBranch;
+        private final LfBranch functionLfBranch; // todo: create two classes: branch function and bus function ?
```

Comment:
Ideally yes, if possible we should have another class with commons abstract for branch or bus monitored factor.